apps/angular/src/app/features/anime/table/anime-table.component.ts

```diff
@@ -0,0 +1,67 @@
+import { ChangeDetectionStrategy, Component, OnDestroy, OnInit } from '@angular/core';
+
+import { AnimeBase } from '@js-camp/core/models/anime/animeBase';
+
+import { map, Observable, shareReplay, Subscription } from 'rxjs';
+
+import { AnimeService } from '../../../../core/services/anime.service';
+
+/** Anime list component. */
+@Component({
+  selector: 'anime-table',
+  templateUrl: './anime-table.component.html',
+  styleUrls: ['./anime-table.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AnimeTableComponent implements OnInit, OnDestroy {
+  /** List of anime. */
+  public readonly animeList$: Observable<readonly AnimeBase[]>;
+
+  /** Anime table subscriptions. */
+  private readonly animeTableSubscriptions = new Subscription();
+
+  /** Is table loading. */
+  public isLoading = true;
+
+  /** Columns to display in table. */
+  public displayedColumns = [
+    'image',
+    'titleEng',
+    'titleJpn',
+    'airedStart',
+    'type',
+    'status',
+  ];
```

Comment:
You can add const assertion to make this array immutable.  https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions

```suggestion
    'titleEng',
    'titleJpn',
    'airedStart',
    'type',
    'status',
  ] as const;
```